pluto_build/bin/config.sh

```diff
@@ -59,16 +62,23 @@ function FGDB_export {
 register 'export' 'gdb' 'export pluto.gdb' FGDB_export
 
 function SHP_export {
+  table=$1
+  geomtype=$2
+  name=${3:-$table}
```

Comment:
Really there is just one arg passed, the table name. Can we simply down from using `$table` and `$name` to just `$table` for everything?

Reply:
yeah those are some copy over issues. Will take a look deleting them once making sure the new build is successful

Reply:
yeah those are some copy over issues. Will take a look deleting them once making sure the new build is successful